src/edit/ScaleHandle.js

```diff
@@ -2,44 +2,46 @@ L.ScaleHandle = L.EditHandle.extend({
       iconSize: [32, 32],
-      iconAnchor: [16, 16]
-    })
+      iconAnchor: [16, 16],
+    }),
   },
 
-  _onHandleDrag: function(e) {
-    var overlay = this._handled,
-		map = overlay._map,
-		edgeMinWidth = overlay.edgeMinWidth,
-		formerLatLng = overlay.getCorner(this._corner),
-		newLatLng = this.getLatLng(),
-		scale = this._calculateScalingFactor(formerLatLng, newLatLng);
+  _onHandleDrag: function() {
+    var overlay = this._handled;
+    var map = overlay._map;
+    var edgeMinWidth = overlay.edgeMinWidth;
+    var formerLatLng = overlay.getCorner(this._corner);
+    var newLatLng = this.getLatLng();
+    var scale = this._calculateScalingFactor(formerLatLng, newLatLng);
 
     /*
      * checks whether the "edgeMinWidth" property is set and tracks the minimum edge length;
      * this enables preventing scaling to zero, but we might also add an overall scale limit
      */
 
     if (!edgeMinWidth) { edgeMinWidth = 50; } /* just in case */
-    var corner1 = map.latLngToLayerPoint(overlay.getCorner(0)),
-		corner2 = map.latLngToLayerPoint(overlay.getCorner(1)),
-		w = Math.abs(corner1.x - corner2.x),
-		h = Math.abs(corner1.y - corner2.y),
-		distance = Math.sqrt(w * w + h * h);
+    var corner1 = map.latLngToLayerPoint(overlay.getCorner(0));
+    var corner2 = map.latLngToLayerPoint(overlay.getCorner(1));
```

Comment:
These commas + var always cause trouble. I'm cool with this syntax!

Reply:
@jywarren were switching to ES6 soon so I think either way its ok but its what we decided to go with in #389